mod_ci/controllers.py

```diff
@@ -266,7 +269,12 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
         # Create XML file for test
         file_name = '{name}.xml'.format(name=category.name)
         single_test = etree.Element('tests')
+        check_write = False
         for regression_test in category.regression_tests:
+            if regression_test.id not in regression_ids:
+                break
+            else:
+                check_write = True
```

Comment:
If you do a break/continue on the if condition, there's no need for an else... You can just set `check_write` to True.